### DISCUSSION TITLE
How to clone my voice and use it with piper?

### DISCUSSION THREAD

**Discussion by siddjain** [rhasspy/piper#581] [Q&A]
- Can anyone tell me if its possible to clone my voice and then use with Piper? How?
- What TTS algorithm does Piper use?

**Comment by king-dahmanus** [comment]
Hello. Yes, you can train your own voice, but you will have to do some
steps first. Piper uses a neural network based DTS engine. First, you have
to record a data set, containing audio and it's text transcription. Then
you have to train the model using your own GPU or a rent GPU like the ones
available on Google collab. Do note that your recordings have to be
consistent and of high quality in order for the model to be of appropriate
good quality sorry, I am using voice dictation on android, so I can't
clarify further, and what I'm typing might not be very accurate. Thanks and
regards.
